components/Feature/VulnerabilitiesGrid/ResourceCard/index.js

```diff
@@ -33,7 +32,8 @@ const ResourceCard = ({
   const distributionElement =  tags.filter(t => HIDDEN_TAGS.includes(t)).join(", ")
   const tagsElement = tags.filter(t => !HIDDEN_TAGS.includes(t)).map(item=> (<span key={"tags-"+item} className={css.tags}>{trimLength(item, 20)}</span>))
   const snapshot = (customerId != undefined) ? true : false
-  const foodResource = (foodNeedsId == 'food-needs') ? true : false
+  const summaryDataExists = ( telephone != undefined || distance != 100 || currentProvision != undefined || openingTimes != undefined) ? true : false
```

Comment:
My js-fu is weak, but from playing in a [REPL](https://repl.it/languages/javascript) I think testing for `undefined` doesn't work with this syntax?

Are we actually checking for [undefined, or null](https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript)?

Reply:
good shout, i think it's null. I wonder why it still works